<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-88179-2"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-88179-2');
</script>

        <link rel="stylesheet" href="style/base.css" type="text/css" />
    </head>
    <body style="margin: 0">
        <div class="jumbotron jumbotron-fluid">
             <div class="container">
               <img src="images/proteore.png/" width="200px">
               <font size="+2"><strong>Welcome to ProteoRE Galaxy !</strong></font>
            </div>
         </div>


<!--
     <div class="container" style="text-align:justify">
            <div class="jumbotron">
                <table style="width:100%">
                    <tr>
                        <th><img src="images/red_warning.png" width="80px"></th>
                        <th style="text-align:center"><font size="3"><strong>Proteore.org will be unavailable for maintenance purpose from Sunday , March 31th to Wednesday , April 3d.<br/>
                            Thank you for your understanding. </strong></font></th>
                        <th><img src="images/red_warning.png" width="80px"></th>
                    </tr>
                </table>
      </div>
-->
        <div class="container" style="text-align:justify">
            <p class="lead">
        <h2>What is ProteoRE ?</h2>
                ProteoRE (Proteomics Research Environment) is a Galaxy-based platform for the interpretation and the exploration of proteomics and transcriptomics data in biomedical research; it allows researchers to apply a large range of dedicated bioinformatics tools and data analysis workflows on their data, share their analyses with others, and enable tiers to reuse the same analysis while keeping tracks of the overall process. The ProteoRE 2.1 version now comprises <b>20 tools</b> organized into four subsections: i) data manipulation and visualization; ii) get features/annotation; iii ) functional analysis; and iv) pathway analysis along with graphical representation. This version also includes Galaxy generic tools and some tools from the GalaxyP project that should be of great help for (re)processing MS-based proteomics data (e.g. peptide/protein identification). 
		<br/>
        <BR>
	<h2>ProteoRE new release (version 2.1 – 31th January 2020)</h2>
	<BR>
            <h3>Update/Enhancement:</h3>
                <ul>
                <li>Tools panel was reorganized (now four sections) and some tools were renamed to make them more explicit (thanks to ProteoRE’s users feedback)</li>
		<li>Expression data source files from HPA (and related tools) have been fully updated</li> 
		<li>Genome wide annotation source files for human, mouse and rat have been updated for all functional analysis (GO) tools</li>  
		<li><b>ID converter</b>: gene name symbol has been included and mapping files updated accordingly for <i>Homo sapiens, Mus musculus, Rattus norvegicus</i> (UniProt release 18/12/2019).</li>
                <li><b>Venn diagram [JVenn]</b>: : bug due to one extra-line counting is now fixed</li>
                <li><b>KEGG pathways mapping and visualization [PathView]</b>: fixed bug associated with KEGG map visualization</li></b>
		</ul>
             <h3>New tools:</h3>
		<ul>
                <li><b><a target="_blank" href="http://www.proteore.org/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fproteore%2Fproteore_prot_features_3orga%2Fprot_features_3orga%2F2020.01.23">Add protein features [UniProt, Human, Mouse, Rat]</a></b>: This tool retrieves protein features (e.g. subcell location, topological domain, pathway, disease) from the UniProt database (Reference Proteome) to annotate your protein IDs list</li>
                <li><b><a target="_blank" href="http://www.proteore.org/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fvolcanoplot%2Fvolcanoplot%2F0.0.3">Volcano plot</a></b>: this tool allows to draw a Volcano Plot and to add labels. Very useful for differential analysis to quickly visualize genes/proteins and to identify the most meaningful changes between two conditions</li>
                <li><b><a target="_blank" href="http://www.proteore.org/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fproteore%2Fproteore_go_terms_enrich_comparison%2Fgo_terms_enrich_comparison%2F2019.12.13.1">Biological themes comparison (Human, Mouse, Rat)[clusterProfiler]</a></b>: this tool performs biological theme comparison by computing enriched functional categories between different protein/gene lists (e.g. in different conditions)</li>
		</ul>
		
              <h5><a target="_blank" href="http://www.proteore.org/static/previous_releases_notes.html">ProteoRE previous Release notes</a></h5>
        <BR>
		<h3>How to access ProteoRE ?</h3>
                This public site is meant for academic usage;  for commercial usage, <a href="mailto:contact@proteore.org">contact us</a>). ProteoRE can currently be used as:  
		<li><b>Unregistered user</b>: you can analyze your data (e.g. proteomics identification results in tab format) using the available tools. Allocated storage space is limited to 20 Mb and data/history/analyses are not conserved between sessions.</li>
                <li><b>Registered user</b> (password-protected access): we highly recommend to create a user account to keep track of and record the analyses and workflows created. Up to 20 Gb storage space is offered to registered users, along with access to shared resources (data, histories, workflows..). To create an account: click on "Account registration or login" on the main menu bar (upper part of the ProteoRE central panel), click on "registration" and fill out the form. Once the form is submitted, your account is created.</li>
        <BR>
		<h3>Tutorials and training materials:</h3>
                Training materials for ProteoRE are available via the Galaxy Training Network: 
		<ul>
                <li><a href="https://training.galaxyproject.org/training-material/topics/proteomics/tutorials/proteome_annotation/tutorial.html" target="_blank"><b> Annotating a protein list identified by LC-MS/MS experiment</b></a></li>
                <li><a href="https://training.galaxyproject.org/training-material/topics/proteomics/tutorials/biomarker_selection/tutorial.html" target="_blank"><b>Strategy to select biomarker candidates</b></a></li>
		</ul>
	<BR>
		<h3>How to contribute? </h3>
		<li><b>As a developper/bioinformatician</b>: ProteoRE welcomes new development via GitHub, please <a href="https://github.com/ifb-git/ProteoRE" target="_blank">read here</a></li>
                <li><b>As a biologist/clinician</b>: ProteoRE is designed for and with biologists/clinicians on the basis of case study using real proteomics data; ProteoRE is constantly evolving thanks to your feedbacks and we are grateful to all users for their suggestion that improve ProteoRE usability; for any specific need, wish, idea, do not hesitate to <a href="mailto:contact@proteore.org">contact@proteore.org</a>contact us.</li>
	<BR>
		<h3>ProteoRE events: <a href="http://www.proteore.org/static/communication.html" target="_blank">training, communication, publication and related links</a></h3>
	<BR>
		<h4>Data sources: <a href="http://www.proteore.org/static/data_source.html" target="_blank">public resources used by ProteoRE's tools</a></h4> 
	<BR>	
		<h4>Support, feedbacks, contribution: <a href="mailto:contact@proteore.org">contact@proteore.org</a></h4>	
	<BR>
	</p>
            <footer>
                <h3>Fundings</h3>
                ProteoRE project is a joint national effort between the <a target="_blank" class="reference" href="http://www.france-bioinformatique.fr/en">French bioinformatics Institute (IFB)</a> and the <a target="_blank" class="reference" href="http://www.profiproteomics.fr/">Proteomics French Infrastructure (ProFI)</a> and is funded by the <a target="_blank" class="reference" href="http://www.agence-nationale-recherche.fr/en/">French Research Agency</a> (ANR-11-INBS-0013).<br/>
                <BR>
                <img src="images/ifb.png/" width="100px">
                <img src="images/profi.png/" width="100px">
                <img src="images/anr.png/" width="100px"><br/>
                <BR>
                <h3>Project team</h3>
                Lien Nguyen, David Christiany (software engineers), Florence Combes (bioinformatician), Virginie Brun, Maud Lacombe, Marianne Tardif & Benoit Gilquin (use-case & beta-testing), Valentin Loux (Bioinformatician , platform admin), Yves Vandenbrouck (Project manager – <a href="mailto:contact@proteore.org">contact@proteore.org</a>)

                The <a target="_blank" class="reference" href="http://galaxyproject.org/">Galaxy Project</a>
                is supported in part by <a target="_blank" class="reference" href="http://www.genome.gov">NHGRI</a>,
                <a target="_blank" class="reference" href="http://www.nsf.gov">NSF</a>,
                <a target="_blank" class="reference" href="http://www.huck.psu.edu">The Huck Institutes of the Life Sciences</a>,
                <a target="_blank" class="reference" href="http://www.ics.psu.edu">The Institute for CyberScience at Penn State</a>,
                and <a target="_blank" class="reference" href="http://www.jhu.edu/">Johns Hopkins University</a>.<br/>
                <BR>
                <BR>
            </footer>
        </div>
    </body>
</html>
